src/Tingle.AspNetCore.JsonPatch/README.md

```diff
@@ -1,131 +1,110 @@
 
 ```cs
-class Customer
-{
-    public string Name { get; set; }
-    public List<Order> Orders { get; set; }
-    public string AlternateName { get; set; }
-}
+var builder = WebApplication.CreateBuilder(args);
 
-class Order
-{
-    public string Item { get; set; }
-    public int Quantity { get; set;}
-}
-```
+builder.Services.AddControllers()
+                .AddJsonPatch() // would be .AddNewtonsoftJson() in the Newtonsoft.Json equivalent
+                .AddJsonOptions(options => {}); // Add and configure JSON formatters
 
-An instantiated `Customer` object would then look like this:
+var app = builder.Build();
 
-```cs
-{
-    "name": "John",
-    "alternateName": null,
-    "orders":
-    [
-        {
-            "item": "Toy car",
-            "quantity": 1
-        },
-        {
-            "item": "C# In A Nutshell Book",
-            "quantity": 1
-        }
-    ]
-}
-```
+app.UseHttpsRedirection();
 
-A JSON Patch document has an array of operations. Each operation identifies a particular type of change. Examples of such changes include adding an array element or replacing a property value.
+app.UseAuthorization();
 
-Let us create `JsonPatchDocument<Customer>` instance to demonstrate the various patching functionalities we provide.
+app.MapControllers();
 
-```cs
-var patchDoc = new JsonPatchDocument<Customer>();
-// Define operations here...
+app.Run();
 ```
 
-By default, the case transform type is `LowerCase`. Other options available are `UpperCase`, `CamelCase` and `OriginalCase`. These can be set via the constructor of the `JsonPatchDocument<T>`. For our example purposes we'll go with the default casing. Now let us see the supported patch operations.
+## Json Merge Patch
 
-## Add Operation
+ The library helps to deserialize HTTP requests' and responses' JSON body content for merge patch operation. If the merge patch request contains members that appear as null on the target object, those members are added. If the target object contains the member, the value is replaced. Members with null values in the merge patch requests, are removed from the target object (set to null or default).
 
-Let us set the `Name` of the customer and add an object to the end of the `orders` array.
+For example, the following JSON documents represent a resource, a JSON Merge Patch document for the resource, and the result of applying the Patch operations.
 
-```cs
-var order = new Order
-{
-    Item = "Car tracker",
-    Quantity = 10
-};
+### Resource Example
 
-var patchDoc = new JsonPatchDocument<Customer>();
-patchDoc.Add(x => x.Name, "Ben")
-        .Add(y => y.Orders, order);
+```json
+{
+    "name": null,
+    "phone": "+254722000000",
+    "country": "ken"
+}
 ```
 
-## Remove Operation
+### JSON Merge Patch Example
 
-Let us set the `Name` to null and delete `orders[0]`
+```json
+{
+    "name": "Fabrikam",
+    "phone": "+254722000001",
+    "country": null
```

Comment:
How about showing properties that remain unchanged because they are no included in the PatchMerge request body?